pkg/option/config.go

```diff
@@ -1362,6 +1384,19 @@ func (c *DaemonConfig) Populate() {
 	c.ToFQDNsProxyPort = viper.GetInt(ToFQDNsProxyPort)
 	c.ToFQDNsPreCache = viper.GetString(ToFQDNsPreCache)
 
+	// Convert IP strings into net.IPNet types
+	subnets, invalid := ip.ParseCIDRs(viper.GetStringSlice(IPv4PodSubnets))
+	if len(invalid) > 0 {
+		log.Warnf("Config IPv4PodSubnets, ParseCIDR %s failed", invalid)
+	}
+	c.IPv4PodSubnets = subnets
+
+	subnets, invalid = ip.ParseCIDRs(viper.GetStringSlice(IPv6PodSubnets))
+	if len(invalid) > 0 {
+		log.Warnf("Config IPv6PodSubnets, ParseCIDR %s failed", invalid)
```

Comment:
nit: `WithFields()`

Reply:
Using withfields() now logs output from garbage input,

level=warning msg="IPv4PodSubnets parameter can not be parsed." IP="[a.b.10.1/24 10.56.1.1/344]" subsys=config